index.html

```diff
@@ -0,0 +1,41 @@
+<!doctype html>
+<html lang="es">
+
+<head>
+    <meta charset="utf-8">
+    <title>My notes Application</title>
+    <link rel="stylesheet" href="sanitizer.css">
+    <link href="style.css" rel="stylesheet" />
+</head>
+
+<body>
+    <header>
+        <h1>My Notes Application</h1>
+    </header>
+    <div class="main_container">
+        <div class="form__wrapper">
+            <h2 class="form__heading">Create a new note</h2>
+            <br>
+            <form class="form">
+                <label class="form__field__title">Title:</label>
+                <input type="text" class="title__input" placeholder="Enter note title">
+                <label class="form__field__title">Content:</label>
+                <textarea class="content__input" rows="6" cols="4" placeholder="Enter note content"></textarea> <br>
+                <button type="submit" class="form_button">Add Note</button>
+            </form>
+        </div>
+        <div class="sidebar">
+            <h4 class="sidebar__title">Resources</h4>
+            <ul>
+                <li><p><a href="github.com">MDN Web docs</a></p></li>
+                <li><p><a href="css.com">css-tricks</a></p></li>
+                <li><p><a href="github.com">My Github</a></p></li>
+            </ul>
+        </div>
+    </div>
+    <footer>
+        <h5 class="footer__text">© 2024 My Notes Application</h5>
```

Comment:
It's not necessary to use heading at this level. Tags p or span should be enough.